README.md

```diff
@@ -1,6 +1,19 @@
 # Stedi EDI Bootstrap
 
-This repo contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one possible way to interact with Stedi's Products and APIs to achieve a typical EDI workload; your implementation may include some or all of these products depending on your particular systems and requirements.
+This repository contains an end-to-end configuration for building a full X12 EDI system using Stedi products. This implementation demonstrates one way to build an integration for the common read and write EDI use case. Your solution may differ, depending on your systems and requirements. We encourage you to [contact us](mailto:support@stedi.com) as you get started with Stedi, so we can help you design a custom integration.
+
+
+# Read and Write Workflow 
+
+The bootstrap uses the following process to handle incoming EDI:
+
+1. An example trading partner uses [Stedi SFTP](https://www.stedi.com/products/sftp) to send EDI documents to a [Stedi Bucket](https://www.stedi.com/products/buckets).
```

Comment:
The product is called Stedi Buckets, the resource is called a Stedi bucket. Which reminds me, I need to add that to the style guide. :-)